tests/ui/feature-gates/test-listing-format-json.rs

```diff
@@ -0,0 +1,18 @@
+// no-prefer-dynamic
+// compile-flags: --test
+// run-flags: --list --format json -Zunstable-options
+// run-fail
+// check-run-results
+// ignore-nightly
```

Comment:
Are we checking this on the stable builder today? Or does that not run tests - I forget.

Reply:
It should be running the full tests, so yes it should be executed in theory.